### ISSUE TITLE
Veyon service does not start on login/computer startup in kubuntu 16.04

### ISSUE THREAD

**Issue by jcoope13** [veyon/veyon#305]
Hello!  I can get veyon set up and running properly on both my linux and windows machines but the only problem I have is that on the linux machines, the veyon-service program does not run as a system service and instead has to be started manually.

I've tried adding it to system startup programs in the system settings to no avail.

I've also tried adding it as a service in systemd with:

file: /lib/systemd/system/veyon.service
============start file==============
[Unit]
Description=Veyon Service

[Service]
ExecStart=/usr/bin/veyon-service

[Install]
WantedBy=multi-user.target
===========end file===============

and then enabling it to run at boot with:

sudo systemctl enable veyon

I can start it with

sudo systemctl start veyon

and stop it with

sudo systemctl stop veyon

Problem: when I run it this way, I cannot connect to these computers at all, the icons in the master are red.   I can see the service running when I use "top" but I cannot access it like if I were to simply run it via terminal "sudo veyon-service".

**Comment by jcoope13** [comment]
I should add that I've tried:
sudo veyon-ctl service register
and the service still doesn't start after a reboot but even without running that once I can start it manually with
sudo veyon-ctl service start

**Comment by tobydox** [comment]
Can you try Veyon 4.1 alpha? It has builtin systemd support and should not require any custom service files  :-)

**Comment by jcoope13** [comment]
I will when I get time to reinstall on one of the target machines.

**Comment by hunterd** [comment]
I tried with the Veyon 4.1 alpha.
I'm using Lubuntu 18.04 x64.

To allow veyon-configurator to write files in the appropriate folder I opened it with:
`pksudo veyon-configurator`

I found that the service file already exists at:
`/lib/systemd/system/veyon-service.service`

its content:
`[Unit]
Description=Veyon Service
After=network-online.target dbus.service systemd-logind.service
Wants=network-online.target
Requires=dbus.service systemd-logind.service

[Service]
ExecStart=/usr/bin/veyon-service
Type=simple
Restart=always
StartLimitInterval=60
StartLimitBurst=10

[Install]
WantedBy=multi-user.target
`

So I restarted my computer to see if it worked properly. Halas it didn't.
I'm checked service status:
`sudo systemctl status veyon-service.service`

it returned:
`● veyon-service.service - Veyon Service
   Loaded: loaded (/lib/systemd/system/veyon-service.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2018-05-28 12:41:59 +04; 5min ago
 Main PID: 642 (veyon-service)
    Tasks: 2 (limit: 1085)
   CGroup: /system.slice/veyon-service.service
           └─642 /usr/bin/veyon-service

mai 28 12:41:59 pc-lubuntu-18-04 systemd[1]: Started Veyon Service.
mai 28 12:42:01 pc-lubuntu-18-04 veyon-service[642]: Adding plugin search path "/usr/lib/x86_64-linux-gnu/veyon"
mai 28 12:42:01 pc-lubuntu-18-04 veyon-service[642]: PluginManager: discovered plugin "LinuxPlatformPlugin" at "/usr/lib/x86_64-linux-gnu/veyon/linux-platform.so"
`

So I restarted the service then checked the response:
`sudo systemctl stop veyon-service.service
sudo systemctl start veyon-service.service
sudo systemctl status veyon-service.service
`
It returned:
`● veyon-service.service - Veyon Service
   Loaded: loaded (/lib/systemd/system/veyon-service.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2018-05-28 12:52:29 +04; 6min ago
 Main PID: 1437 (veyon-service)
    Tasks: 10 (limit: 1085)
   CGroup: /system.slice/veyon-service.service
           ├─1437 /usr/bin/veyon-service
           ├─1439 /usr/bin/veyon-server
           └─1440 /usr/bin/veyon-worker {8e997d84-ebb9-430f-8f72-d45d9821963d}

mai 28 12:52:29 pc-lubuntu-18-04 systemd[1]: Started Veyon Service.
mai 28 12:52:29 pc-lubuntu-18-04 veyon-service[1437]: Adding plugin search path "/usr/lib/x86_64-linux-gnu/veyon"
mai 28 12:52:29 pc-lubuntu-18-04 veyon-service[1437]: PluginManager: discovered plugin "LinuxPlatformPlugin" at "/usr/lib/x86_64-linux-gnu/veyon/linux-platform.so"
mai 28 12:52:29 pc-lubuntu-18-04 veyon-service[1437]: 2018-05-28T12:52:29.428: [INFO] Starting server for new session "/org/freedesktop/login1/session/c2" with display ":0" at seat "/org/freedesktop/login1/seat/seat0"
`

This time I can see the remote machine but each time I restart the computer the problem reappears.
